frontend/src/app/main/ui/workspace/plugins.cljs

```diff
@@ -204,6 +222,7 @@
              [:& plugin-entry {:key (dm/str "plugin-" idx)
                                :index idx
                                :manifest manifest
+                               :user-can-edit? user-can-edit?
```

Comment:
Don't use `?` on component props, props is a data structure

Reply:
Fixed